apps/user-portal/src/api/consents.ts

```diff
@@ -38,6 +33,9 @@ const httpClient = AxiosHttpClient.getInstance();
  * @return {Promise<any>} A promise containing the response.
  */
 export const fetchConsentedApps = (state: ConsentState): Promise<any> => {
+    const userName = AuthenticateSessionUtil.getSessionParameter(AuthenticateUserKeys.USERNAME).split("@");
+    userName.pop();
```

Comment:
Validate before `pop()` the last element whether a `@` is there in the `sting`. Ans then do the split